src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/IssueCreateDialog.kt

```diff
@@ -44,7 +45,6 @@ class IssueCreateDialog(
 
     fun selectProject(projectName: String) = form.waitForRefresh(Supplier {
         projectField.select(projectName)
-        waitUntilSummaryIsFocused()
```

Comment:
Huh, this is deleted, but tests didn't go red. Maybe this waiting wasn't necessary, but let's change that in a separate PR.

Reply:
It started to fail on this lane after the accessibility changes tho when I removed it the previous versions seems to work just fine. 

I'll follow your advice and prepare a separate PR.

Reply:
Often tests can go green without waits. It could just become flaky.